### PR TITLE
[stable/contour] add chart

### PR DESCRIPTION
**Pull request by poidag-zz**
Signed-off-by: pegrant <9246508+pickledrick@users.noreply.github.com>

#### What this PR does / why we need it:
Adds a helm chart for [Project Contour](https://projectcontour.io/)

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

**Comment by pickledrick** [comment]
Not sure why the linting is failing or if there is something I have to do to make it work as this is a new chart and it seems to be targeting master branch?

**Comment by pickledrick** [comment]
Hi @jlegrone, @mattfarina,

I'm getting a 404 on lint tests wrt not finding an owner under my name. I have added myself to the `chart.yaml` and in the `OWNERS` file.

Is there anywhere else this needs to be added?

**Comment by pickledrick** [comment]
Hi @unguiculus 

I recall you were very helpful in a previous PR to this repo. Are you able to take a look and see what the problem is with the owners file? I suspect it's because I'm adding a new chart?

**Comment by pickledrick** [comment]
Hi @maorfr 

I see you have some issues open wrt calls for maintainers. Are you able to help with this PR?

**Comment by prydonius** [comment]
@pickledrick I have sent you an invite to become a collaborator of this repo. Is there anyone else from the Contour team we could add as a maintainer? With more than one maintainer, you'll be able to merge future changes between you, without the need for approval from a repo maintainer.

File: stable/contour/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+description: Contour Ingress controller for Kubernetes
+name: contour
+version: 0.1.0
+home: https://projectcontour.io
+keywords:
+  - ingress
+  - envoy
+  - contour
+maintainers:
+  - name: Peter Grant
```
Comment (prydonius) [inline stable/contour/Chart.yaml]: name needs to be your GitHub username

```suggestion
  - name: pickledrick
```

**Comment by pickledrick** [comment]
Hi @prydonius,

Are you able to take a look at this or suggest someone who should review for merge?

**Comment by prydonius** [comment]
@pickledrick sorry for the delay, are you able to add someone else in the OWNERS file who could help you merge changes going forward?

**Comment by pickledrick** [comment]
Hi @prydonius I have updated.

cc: @boluisa I have added you to `stable/contour/OWNERS`

**Comment by prydonius** [comment]
Great, thanks @pickledrick, I sent @boluisa an invite to collaborate on the repo.

/lgtm

**Comment by boluisa** [comment]
> Great, thanks @pickledrick, I sent @boluisa an invite to collaborate on the repo.
> 
> /lgtm

invite Accepted. exciting

**Pull request merged**